src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -53,6 +53,7 @@ void Analyzer::Display() {
                                                       m_logger);
         m_type = m_manager->GetAnalysisType();
         Calculate();
+        PrepareData();
```

Comment:
Call this before `Calculate()`.

Reply:
I think calling it before `Calculate()` segfaults it because `PrepareData()` loads the graph data that requires feedforwards. What would be the best way of resolving this? Should we have a separate method solely to load graph data?

Reply:
I think it crashed when doing it in the other order.

Reply:
Yes I think we should have a separate function to load plot data.